File: trabajos en javascript/interaccion usuario.js
var prompt = require('prompt-sync')();

let name =prompt("ingrese su nombre: ")
let lastName =prompt("ingrese su apellido: ")
let prof =prompt("ingrese su profesion: ")
let birth =prompt ("ingrese su año de nacimiento: ")
let age = (2025-birth)
console.log("El (la) usuario se llama ", name, lastName, "es un (a) ", prof, "y su edad es de ", age)

File: trabajos en GO/interaccion_usuario.go
package main

import "fmt"

func main() {
	var nombre string
	fmt.Print("Ingrese su nombre: ")
	fmt.Scan(&nombre)
	var apellido string
	fmt.Print("Ingrese su apellido: ")
	fmt.Scan(&apellido)
	var profesion string
	fmt.Print("Ingrese su profesion: ")
	fmt.Scan(&profesion)
	var nacimiento int
	fmt.Print("Ingrese su año de nacimiento: ")
	fmt.Scan(&nacimiento)
	var edad int
	edad = 2025 - nacimiento
	fmt.Println("Hola", nombre, apellido, "de", edad, "años, eres", profesion)
}
